<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarUi</class>
 <widget class="QWidget" name="RadarUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadarUi</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="6">
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Line_Width</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QDoubleSpinBox" name="ColorA">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.400000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Color:B</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Color:R</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="ColorB">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.600000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QDoubleSpinBox" name="lineWidth">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Color:A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Color:G</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="ColorG">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.600000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="DisplayList">
      <item>
       <property name="text">
        <string>surface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>line</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="ColorR">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>DisplayMode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
